pkg/tools/opa/policies/curr_token_signed.opa

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Cisco and/or its affiliates.
```

Comment:
Please use `.rego` extension for OPA files - this extension is supported by default by Goland OPA plugin.

Reply:
Thats good catch. Will do :)